libs/core/src/components/pds-text/docs/pds-text.mdx

```diff
@@ -0,0 +1,107 @@
+import {DocArgsTable, DocCanvas} from '@pine-ds/doc-components';
+import { components } from '../../../../dist/docs.json';
+
+# Text
+
+The Text component allows for a choice of common semantic tags used for headings or body text. This tag can be nested so `em` and `strong` can be used as well.
+
+## Properties
+
+<DocArgsTable componentName="pds-text" docSource={components} />
+
+## Default
+
+A `tag` is required for this component. Each semantic tag comes with default styling.
```

Comment:
Because `tag` is required, it will require the devs to know when to use the right tag. As a followup, we may need Accessibility Best Practice docs, with this being one of the section

Reply:
Added Guidelines and References sections.